save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/calendar/ReactCalendarBuilder.kt

```diff
@@ -0,0 +1,26 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package com.saveourtool.save.frontend.externals.calendar
+
+import org.w3c.dom.events.Event
+import react.ChildrenBuilder
+import react.react
+import kotlin.js.Date
+
+/**
+ * @param onChange
+ * @param handler
+ */
+fun ChildrenBuilder.calendar(
```

Comment:
![](https://ememetees.com/data/gfx/pictures/medium/4/2/2224_5.jpg)

Reply:
We should have English calendar

Reply:
Added :)